sequencing/vcf_to_plink/Readme.md

```diff
@@ -1,15 +1,13 @@
-# vcf_to_plink (DNAnexus Platform App)
+# Convert VCF file to PLINK file (DNAnexus Platform App)
 
-Uses PLINK 1.9 to convert VCF files to PLINK bed/bim/fam files.
```

Comment:
"What does this applet do?" -> "What does this app do?"
I am assuming that all applets will eventually be published as apps, for the paper's publication